images/bootstrap/Dockerfile

```diff
@@ -14,6 +14,38 @@
 # limitations under the License.
 
 # Includes basic workspace setup, with gcloud and a bootstrap runner
+FROM fedora:34 AS builder
```

Comment:
time has passed, 35 may be better?

Reply:
yes fedora 35 has been available for a while now - updated.

Reply:
updated to fedora 36